py/head.py

```diff
@@ -2,18 +2,51 @@
 from dominate.tags import *
 
 def _head():
-    """Returns the generated HTML header. TODO: edit this description
-
-    TODO: A description here, according to Google's Python style guide https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings
+    """This function generates a header using tables and returns it as HTML. DO NOT EDIT!
 
     Args:
-        TODO
+        None
 
     Returns:
-        TODO
+        The generated HTML header
```

Comment:
Can we have a more descriptive comment for what this function returns? You don't get any more information out of this description than reading the function name.